duckduckgo/duckduckgo-privacy-extension
# Tracker count updates

**Reviewer:** @laurengarcia @andrey-p 



## Description:
- Update the trackers count to show *all* the blocked tracker requests instead of counting each domain only once
- Consequently, change text from "X Tracker Networks Blocked/Found" to "X Trackers Blocked/Found"


## Steps to test this PR:
1. Build and reload extension
2. Navigate to sites with trackers and verify the count (console logging the trackersCount is also an easy option)
3. Optional: compare number of trackers blocked or found with other tracker blocker extensions - it should be mostly on par or better
 4. When 0 trackers are found, the text will say "0 Trackers Found"


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
